pkg/netutils/netutils.go

```diff
@@ -42,6 +42,28 @@ func EnsureIPAssigned(iface, cidr string) error {
 	return netlink.AddrAdd(link, addr)
 }
 
+// ensure that given IP is not present on a given link
+func EnsureIPUnassigned(iface, cidr string) error {
```

Comment:
if you want to add docstring please start it with name of the function/method/struct. it is in code guidelines